platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/crypto/MerkleCryptoFactory.java

```diff
@@ -46,22 +51,43 @@ public class MerkleCryptoFactory {
 
     private MerkleCryptoFactory() {}
 
+    /**
+     * Setup cryptography. Intended only to be invoked by unit tests.
+     */
+    private static void defaultTestSetup() {
```

Comment:
Again I think it is bad practice to define runtime code as "This is test code". We should move that to test fixtures and create a JUnit extension. While that can not all be part of one PR we should create issues for that but not name the methods like that.

Reply:
Let's discuss this when we are both next online. I'm not sure how to write a test fixture that can be forced to run before all unit tests in the repo.

Reply:
Renamed to `init()` like suggested for the config holder. Let's discuss next steps together the next time we are both online.